extension-manifest-v3/scripts/build.js

```diff
@@ -285,6 +280,19 @@ const buildPromise = build({
         minifyInternalExports: false,
         entryFileNames: '[name].js',
         assetFileNames: 'assets/[name].[ext]',
+        sanitizeFileName: (name) => {
+          name = name
+            .replace(/[\x00?*]+/g, '_') // eslint-disable-line no-control-regex
```

Comment:
This is taken from the default behavior:
https://rollupjs.org/configuration-options/#output-sanitizefilename